torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15630,10 +15630,10 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            )
            ),
     OpInfo('logcumsumexp',
-           dtypes=floating_types_and(torch.bfloat16),
-           dtypesIfCUDA=floating_types_and(torch.half, torch.bfloat16),
-           backward_dtypes=floating_types_and(torch.bfloat16),
-           backward_dtypesIfCUDA=floating_types_and(torch.bfloat16),
+           dtypes=floating_and_complex_types_and(torch.bfloat16),
```

Comment:
Shouldn't the sample inputs be changed to also generate complex numbers?

Reply:
The sample inputs generated in `sample_inputs_logcumsumexp` using `make_tensor` using dtype that includes complex as indicated by `floating_and_complex_types_and`. Should it generate the complex numbers automatically?